airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/json/Jsons.kt

```diff
@@ -419,11 +419,11 @@ object Jsons {
      * the class name can at least help narrow down the problem, without leaking
      * potentially-sensitive information. </snip...>
      */
-    private fun <T : Any> handleDeserThrowable(t: Throwable): Optional<T> {
+    private fun <T : Any> handleDeserThrowable(throwable: Throwable): Optional<T> {
         // Manually build the stacktrace, excluding the top-level exception object
         // so that we don't accidentally include the exception message.
         // Otherwise we could just do ExceptionUtils.getStackTrace(t).
-        var t: Throwable? = t
+        var t: Throwable? = throwable
```

Comment:
`t` should not exist at all if the function arg is legitimately not nullable

Reply:
I actually need a new object, because it's being reassigned line 434. But I can make it non-nullable